security/providers/oidc/src/main/java/module-info.java

```diff
@@ -22,7 +22,7 @@
  */
 @Feature(value = "OIDC",
         description = "Security provider for Open ID Connect authentication",
-        in = {HelidonFlavor.SE, HelidonFlavor.MP, HelidonFlavor.NIMA},
+        in = {HelidonFlavor.MP, HelidonFlavor.NIMA},
```

Comment:
This is not an MP feature. There is a `microprofile/oidc` module that adds this feature for Helidon MP